src/interpreter_funcs.rs

```diff
@@ -169,18 +170,45 @@ impl Func for FuncImplLaplacianSmoothing {
             iterations_unclamped
         };
 
-        let value = laplacian_smoothing(geometry, cast_u8(iterations));
+        let value = mesh_smoothing::laplacian_smoothing(geometry, cast_u8(iterations));
         Value::Geometry(Arc::new(value))
     }
 }
 
+pub struct FuncImplSeparateIsolatedMeshes;
+impl Func for FuncImplSeparateIsolatedMeshes {
+    fn flags(&self) -> FuncFlags {
+        FuncFlags::empty()
```

Comment:
If no flag is provided let's omit the implementation altogether - there is a default impl in the trait.

Reply:
I suggest to resolve the purity of this later. We can leave a `FIXME`

Reply:
I agree with adding the task or FIXME to do this, but that is not what this comment was about. You can remove the whole function so that the default is used.